rust/stackablectl/src/cli/mod.rs

```diff
@@ -290,3 +291,27 @@ fn get_files(default_file: &str, env_key: &str) -> Result<Vec<PathOrUrl>, PathOr
 
     Ok(files)
 }
+
+#[derive(Clone, Debug, Default, ValueEnum)]
+pub enum ChartSourceTypeArg {
+    /// OCI registry
+    #[default]
+    OCI,
+
+    /// Nexus repositories: resolution (dev, test, stable) is based on the version and thus will be operator-specific
+    Repo,
+}
+
+impl From<ChartSourceTypeArg> for ChartSourceType {
```

Comment:
question: Why do we need this mapping at all? Can't we use `ChartSourceType` directly in the CLI?

Reply:
We need the derive macro `ValueEnum` and that would mean including the clap crates in stackable-cockpit. What do you think?

Reply:
As agreed, we'll leave this as it is and add a comment in code as to the reason.

Reply:
See 1e6947e